tests/unit/test_charm.py

```diff
@@ -153,7 +158,16 @@ def test_wp_config(
     assert: generated wp-config.php should be valid.
     """
 
-    def in_same_line(content, *matches):
+    def in_same_line(content: str, *matches: str):
+        """Check if matches belong in same line within a content.
```

Comment:
Perhaps I would rephrase to
`Check if all matches are found within the same content line`